### PR TITLE
chore: Enables gofmt linting

### PR DESCRIPTION
**Pull request by gruyaume**
# Description

This PR enables the go fmt lint check and makes the necessary change to satisfy it.

### CODE REVIEW

**Comment by gab-arrobo** [comment]
ok to test

File: util/path_debug.go [hunk 1]
```diff
@@ -3,7 +3,8 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-//+build debug
+//go:build debug
+// +build debug
```
Comment (gab-arrobo) [inline util/path_debug.go:7]: Is this change needed? In my setup, lint successfully completes without this change.

Comment (gruyaume) [reply util/path_debug.go:7]: This change was done by `go fmt ./...`, it was not a manual change

Comment (gab-arrobo) [reply util/path_debug.go:7]: Ok

**Review by gab-arrobo — APPROVED** [review]
+1 (tested changes locally)
